packages/core/vue/src/useWallet.ts

```diff
@@ -109,36 +104,29 @@ export const createWalletStore = ({
 
     // Create a wallet dictionary keyed by their name.
     const walletsByName = computed(() => {
-        return wallets.reduce<Record<WalletName, Wallet>>((walletsByName, wallet) => {
+        return wallets.reduce<Record<WalletName, Adapter>>((walletsByName, wallet) => {
             walletsByName[wallet.name] = wallet;
             return walletsByName;
         }, {});
     });
 
-    // Update the wallet and adapter based on the wallet provider.
+    // Update the wallet based on the wallet provider.
     watch(
         name,
         (): void => {
             const wallet = walletsByName.value?.[name.value as WalletName] ?? null;
-            const adapter = wallet && wallet.adapter;
-            if (adapter) {
+            if (wallet) {
                 setState({
                     wallet,
-                    adapter,
-                    ready: false,
-                    publicKey: adapter.publicKey,
-                    connected: adapter.connected,
+                    ready: wallet.readyState == WalletReadyState.Installed,
```

Comment:
Yeah, this isn't a valid assertion. The ready state of a `Loadable` wallet is still "ready", as in it's ready to be loaded on demand. But in general, as with the React package, we want to surface this through the state because the distinction matters for building more intuitive UIs, e.g. https://github.com/solana-labs/wallet-adapter/blob/16fb5cc395e9974a91297549feaf7694191c7093/packages/ui/react-ui/src/WalletModal.tsx#L23-L48

Reply:
This has been changed to expose the raw type